components/sup/src/manager/service.rs

```diff
@@ -276,7 +277,8 @@ impl Service {
                      last_election_status: ElectionStatus::None,
                      user_config_updated: false,
                      needs_restart: false,
-                     initialization_state: InitializationState::Uninitialized,
+                     initialization_state:
+                         Arc::new(RwLock::new(InitializationState::Uninitialized)),
```

Comment:
:thinking: Wonder if it'd be worth implementing `Default` for `InitializationState`...